common-graphql/src/main/scala/queries/common/ConstraintGroupQueriesGQL.scala

```diff
@@ -59,6 +59,24 @@ object ConstraintGroupQueriesGQL {
           	    microseconds
           	  }
           	}
+            scienceConfiguration {
+              gmosNorthLongSlit {
+                filter
+                disperser
+                fpu
+                slitWidth {
+                  microarcseconds
+                }
+              }
+              gmosSouthLongSlit {
+                filter
+                disperser
+                fpu
+                slitWidth {
+                  microarcseconds
+                }
+              }
+            }
```

Comment:
Do we actually need this information for the constraints tab? Or, is it just here because of the changes to ObsSummary?

Reply:
We are only showing the disperser, so it could be simplified at the cost of adding more classes to only hold that field